logic/atrium-logic-common/src/main/kotlin/ch/tutteli/atrium/logic/AnyExpectations.kt

```diff
@@ -9,7 +9,7 @@ import kotlin.reflect.KClass
 /**
  * Collection of assertion functions and builders which are applicable to any type (sometimes `Any?` sometimes `Any`).
  */
-interface AnyAssertions {
+interface AnyExpectations {
```

Comment:
I think we should not do this renaming at the moment. If we don't rename Assertion (and I suggest we don't for the moment) then it makes more sense to call this AnyAssertions as all functions create an `Assertion` or a `SubjectChangerBuilder.ExecutionStep` (and in other files also `FeatureExtractorBuilder.ExecutionStep`)

Could you please revert the changes in the path /logic for now

Reply:
I am sure now that I will keep the name Assertion on the logic level. The main type on the logic level is also AssertionContainer instead of Expect.